airflow/stackable/utils/entrypoint.sh

```diff
@@ -34,7 +34,9 @@ set -euo pipefail
 # The side effect of this is slightly (in the range of 100s of milliseconds) slower load for any
 # binary started and a little memory used for Heap allocated by initialization of libstdc++
 # This overhead is not happening for binaries that already link dynamically libstdc++
-LD_PRELOAD="/usr/lib/$(uname -m)-linux-gnu/libstdc++.so.6"
+# LD_PRELOAD="/usr/lib/$(uname -m)-linux-gnu/libstdc++.so.6"
+# Stackable: this requires the g++ dnf package to be installed in the image.
```

Comment:
That comment can be removed as well and I think then we're good :slightly_smiling_face: Maybe remove the commented line above this one as well.

Reply:
Done. I left the old path for future reference.